### PR TITLE
Add config option to deny region bypass by default

### PR DESCRIPTION
**Pull request by JOO200**
Should be some small thing.
The default value is set to "false" so there is no change to current behavior.

### CODE REVIEW

File: worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/BukkitWorldConfiguration.java [hunk 1]
```diff
@@ -280,6 +280,7 @@ public void loadConfiguration() {
         maxClaimVolume = getInt("regions.max-claim-volume", 30000);
         claimOnlyInsideExistingRegions = getBoolean("regions.claim-only-inside-existing-regions", false);
         boundedLocationFlags = getBoolean("regions.location-flags-only-inside-regions", false);
+        disableDefaultBypass = getBoolean("regions.disable-default-bypass", false);
```
Comment (wizjany) [inline worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/BukkitWorldConfiguration.java:283]:
```suggestion
        disableDefaultBypass = getBoolean("regions.disable-bypass-by-default", false);
```

File: worldguard-core/src/main/java/com/sk89q/worldguard/session/Session.java [hunk 2]
```diff
@@ -103,6 +103,8 @@ public void initialize(LocalPlayer player) {
 
         lastValid = location;
         lastRegionSet = set.getRegions();
+        disableBypass = WorldGuard.getInstance().getPlatform().getGlobalStateManager()
+                .get(player.getWorld()).disableDefaultBypass;
```
Comment (wizjany) [inline worldguard-core/src/main/java/com/sk89q/worldguard/session/Session.java:108]: Would like to have something here like `if (disableBypass) player.printInfo("You are bypassing region protection...")` so the user knows.

**Review by wizjany — CHANGES_REQUESTED** [review]
Small UX changes requested.

Also "Closes #1630"

File: worldguard-core/src/main/java/com/sk89q/worldguard/session/Session.java (hunk 2 above)
Comment (JOO200) [reply worldguard-core/src/main/java/com/sk89q/worldguard/session/Session.java:108]: DisableBypass means you're not bypassing region protections. Your suggested message doesn't make sense

And imo that shouldn't be necessary or another config option. On many servers there's already a spam at login from different plugins, I don't like the idea to add another message to the login spam

Comment (JOO200) [reply worldguard-core/src/main/java/com/sk89q/worldguard/session/Session.java:117]: I added a configurable message for users who can toggle bypass.

**Comment by wizjany** [comment]
Er, just noticed this uses world config instead of global. Does it make sense for the login world to affect a server-wide (session) toggle? e.g. logging on to a world with the bypass off, then teleporting to one with it on, it will remain off. Think it's probably best to just hoist it to the server-wide config instead of per-world.

**Comment by JOO200** [comment]
I agree, server wide config should be good enough. I changed it.

Maybe some world dependent stuff could be nice but that would need some listener stuff for world changes etc. I don't think that we need that atm.

**Review by wizjany — APPROVED** [review]

**Pull request merged**